src/Kernel/CompiledCode.class.st

```diff
@@ -81,8 +81,10 @@ CompiledCode >> accessesField: varIndex [
 { #category : #scanning }
 CompiledCode >> accessesRef: literalVariable [
```

Comment:
Now we can remove this method actually. LiteralVariable can call #hasLiteral: directly.
But maybe it's good to have for consistency. I would at least rename it to #accessesLiteralVariable: (I do not get what Ref means here)

Reply:
yes, it was about getting a nice API on CompiledMethod: read/write and access for each kind of var (low level arg), then read/write/access for Var in general. renaming *Ref to *LiteralVariable is in the backlog.